sequential.py

```diff
@@ -0,0 +1,32 @@
+import re
+import os
+import subprocess
+import datetime
+for i in range(1, 120):
+    fp = open(r'pvc.yaml', 'r')
+    str1 = 'claim' + str(i)
+    replace_data = fp.read().replace('claim', str1)
+    fp.seek(0, 0)
+    fw = open(r'pvc.yaml', 'w')
+    fw.write(replace_data)
+    fw.seek(0, 0)
+    s = datetime.datetime.now()
```

Comment:
create a separate function to determine execution time of function (hint: Google it, hint2: decorators)